sites/plasticsmachinerymanufacturing.com/config/site.js

```diff
@@ -49,17 +49,17 @@ module.exports = {
   contactUs: {
     branding: {
       bgColor: '#164f77',
-      logo: 'https://img.plasticsmachinerymagazine.com/files/base/ebm/pmm/image/static/logo/site_logo.png?h=60',
+      logo: 'https://img.plasticsmachinerymanufacturing.com/files/base/ebm/pmm/image/static/logo/site_logo.png?h=60',
     },
     to: 'contact@plasticsmachinerymagazine.com',
   },
   inquiry: {
     enabled: true,
     directSend: true,
     sendTo: 'sales@plasticsmachinerymagazine.com',
-    sendFrom: 'SecurityInfoWatch.com <noreply@baseplatform.io>',
+    sendFrom: 'PlasticsMachineryManufacturing.com <noreply@baseplatform.io>',
```

Comment:
lol

Reply:
yeah, who knows how long that little mistake was up there